src/ray/object_manager/pull_manager.cc

```diff
@@ -182,70 +166,53 @@ bool PullManager::ActivateNextPullBundleRequest(const Queue &bundles,
     }
   }
 
-  // Update the pointer to the last pull request that we are actively pulling.
-  RAY_CHECK(next_request_it->first > *highest_req_id_being_pulled);
-  *highest_req_id_being_pulled = next_request_it->first;
+  bundles->ActivateBundlePullRequest(next_request_id);
 
   num_active_bundles_ += 1;
   return true;
 }
 
-void PullManager::DeactivatePullBundleRequest(
-    const Queue &bundles,
-    const Queue::iterator &request_it,
-    uint64_t *highest_req_id_being_pulled,
+void PullManager::DeactivateBundlePullRequest(
+    BundlePullRequests *bundles,
```

Comment:
```suggestion
    BundlePullRequests &bundles,
```

(per [style guide](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs))

Reply:
oh, apparently I'm still remembering the old style guide: https://github.com/google/styleguide/commit/7a7a2f510efe7d7fc :)